Blockzilla.xcodeproj/xcshareddata/xcschemes/Focus (Enterprise).xcscheme

```diff
@@ -37,6 +37,10 @@
                BlueprintName = "XCUITest"
                ReferencedContainer = "container:Blockzilla.xcodeproj">
             </BuildableReference>
+            <LocationScenarioReference
+               identifier = "London, England"
+               referenceType = "1">
+            </LocationScenarioReference>
```

Comment:
Ohh, I was playing around with this for prox testing (location simulation), yes this is unintended, I'll remove it, thanks!